casepro/pods/tests/test_base.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
Resolved in https://github.com/rapidpro/casepro/pull/107/commits/176e7372c818fbb24be1205a98871abc90d390c1